### DISCUSSION TITLE
Spring 3 Cauliflower Seeds

### DISCUSSION THREAD

**Discussion by jbressle** [nathan-alden-sr/stardew-valley#46] [Ideas]
If the player has no Cauliflower Seeds on Spring 3, what do you think about having them swing by Joja Mart to purchase? Pierre's is closed because it's Wednesday. 

The guide already has the player going to Clint's which is right near Joja Mart. The player can then swing by the farm to plant, then go down to fish by Leah's.

Just an idea!

Also, thanks so much for the guide, I'm loving it!!

**Comment by nathan-alden-sr** [comment]
Interesting idea. I haven't played Stardew in ages, but if I ever give the run another go, I'll be sure to try out your suggestion.
